MultiOracle.md

```diff
@@ -0,0 +1,450 @@
+nature of the outcomes is used from a multi-oracle support perspective.
+
+If instead the second kind of DLC is being executed, then a special, more intricate procedure detailed below
+must be used to ensure proper execution.
+This procedure differs from the simple enumerated or exact numeric case as it uses the numeric structure of
+related outcomes.
+Specifically distance is used so that outcomes are related if they are sufficiently near each other.
+
+### Design
+
+It turns out that there is an exceedingly large design space of approaches that can be taken to support multiple
+oracles with some acceptable variance.
+It seems to be generally true, however, that a trade-off exists between the precision of guarantees that a procedure
+can make, and the size of the multiplier on the number of CETs required for the DLC.
+This specification chooses to prioritize CET reduction at the expense of precise guarantees so as to more easily
+support more oracles, which should generally counteract some of the imprecision surrounding variance support.
+Although this resulting design is extremely opinionated in some sense, it does leave three parameters unspecified
+to be negotiated or chosen by DLC participants.
+
+The first two parameters are the orders of magnitude of expected and of allowed difference between oracles.
+By expected difference we mean the differences for which support is required by users, and by allowed difference
+we mean the differences for which support is acceptable but not required.
+These parameters will, from now on, be called `minFail` and `maxError` as they also represent the lower bound
+on failure (non-support) and the upper bound on allowed (support) difference/error between oracles.
+When we say that the guarantees made by this proposal are not precise, we mean two things.
+First, that `minFail` is strictly less than `maxError` so that while it is true that all differences less than `minFail` and none
+greater than `maxError` are supported, there are no guarantees made for differences in between these two parameters.
+Second, that `minFail` and `maxError` are required to be powers of `2` and not arbitrary numbers so that in reality, allowed
+variance is only expressed in terms of order of magnitude, which is a somewhat coarse measure.
+These two things together also imply that `minFail` must be at most half of `maxError` so that the gap between `minFail`
+and `maxError` where no strict guarantees can be made is of significant size.
+
+The third parameter offers a minimal remedy to these lack of guarantees.
+It is a boolean flag (`true` or `false`) called `maximizeCoverage`.
+This parameter arises from the fact that there is a decision to be made when covering the `minFail` variance to
+either cover as much as possible (up to `maxError`) or as little as possible, and this decision has no consequence
```

Comment:
`minFail` is less or equal OR less. Unclear by using "up to `maxError`".

Reply:
Not sure what you mean